beep/conversion_schemas/structured_dtypes.yaml

```diff
@@ -1,4 +1,5 @@
 cycles_interpolated:
+  test_time: 'float32'
```

Comment:
@chirranjeevigopal-TRI Just doing some back of the envelope math, we probably need roughly 10ms resolution even after a year of cycling which means being able to represent ~9 digits in the mantissa? `float32` is a 23bit mantissa which is ~7 digits in the mantissa. So I am guessing this needs to be a float64 if we want to preserve full precision? Kind of wasteful, but I can't think of alternative off the bat. Thoughts?

Reply:
:-) I was calculating with a 100ms resolution, figuring that would already suffice even for regen-braking or HPPC, but just changed it to 64bits.